src/SW_Output.c

```diff
@@ -274,6 +274,8 @@ static void sumof_vpd(SW_VEGPROD *v, SW_VEGPROD_OUTPUTS *s, OutKey k, TimeInt do
 
 		default:
 			LogError(LogInfo, LOGERROR, "PGMR: Invalid key in sumof_vpd(%s)", key2str[k]);
+            return; // Exit function prematurely due to error

```

Comment:
Why did the functions `str2key()` and `str2stype()` not obtain a `return; // Exit function prematurely due to error`? I agree it is not strictly -- I was just guessing that you added a premature `return` to every instance of `LogError(LogInfo, LOGERROR, ...)` for consistency?